wagtail/admin/templates/wagtailadmin/pages/create.html

```diff
@@ -6,13 +6,18 @@
 {% block bodyclass %}page-editor create model-{{ content_type.model }}{% endblock %}
 
 {% block content %}
-
     <header class="merged tab-merged">
         {% explorer_breadcrumb parent_page include_self=1 %}
 
         <div class="row">
             <div class="left col9 header-title">
-                <h1>{% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span></h1>
+                <h1>
+                    {% icon "doc-empty-inverse" class_name="header-title-icon" %}
+                    {% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span>
+                </h1>
+                {% if content_type.model_class.get_page_description %}
+                    <h3>{{ content_type.model_class.get_page_description }}</h3>
```

Comment:
This would be better as a `p` or un-semantic `div`. There shouldn’t be two headings one after another like this.